notification-hubs-sdk/src/main/java/com/microsoft/windowsazure/messaging/notificationhubs/InstallationManager.java

```diff
@@ -0,0 +1,7 @@
+package com.microsoft.windowsazure.messaging.notificationhubs;
+
+import com.microsoft.windowsazure.messaging.notificationhubs.async.NotificationHubFuture;
+
+public interface InstallationManager {
+    NotificationHubFuture<String> saveInstallation(Installation installation);

```

Comment:
What is the string being returned here and how is it useful? Some documentation should be needed here

Reply:
Fixed in b5039971c5e48447912617bb42561b6e3cdc88c1